html/login/views/dashboard.php

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Dashboard</title>
+    <link rel="stylesheet" type="text/css" href="../public/style.css">
+</head>
+<body>
+    <h1>Welcome to the Dashboard</h1><br>
+    <a href="../views/login.php">Logout</a>
+</body>
+</html>
```

Comment:
add new line

Reply:
Yes sir, i already resolved and pushed it. Thank you.